app/cdap/components/WrangleHome/Components/OngoingDataExplorations/utils.ts

```diff
@@ -14,22 +14,34 @@
  * the License.
  */
 
-import { ImportDatasetIcon } from '../WrangleCard/iconStore/ImportDatasetIcon';
-import { IMassagedObject } from './types';
+import DataPrepStore from 'components/DataPrep/store';
+import { IEachData, IMassagedObject } from './types';
 
-export const generateDataForExplorationCard = (oldData) => {
+export const generateDataForExplorationCard = (oldData: IEachData[]) => {
```

Comment:
can we rename old data to something more specific - data-driven names?
e.g. existingExplorationCards